app/src/main/java/com/google/android/apps/flexbox/MainActivity.java

```diff
@@ -16,8 +16,6 @@
 
 package com.google.android.apps.flexbox;
 
-import com.google.android.flexbox.FlexboxLayout;
```

Comment:
Could you restore the import order here?
The codebase follows the [AndroidStyle](https://github.com/android/platform_development/blob/master/ide/intellij/codestyles/AndroidStyle.xml)

I'll update the CONTRIBUTING.md to mention that.